controllers/scripts.go

```diff
@@ -61,22 +70,44 @@ ipfs config Addresses.API /ip4/0.0.0.0/tcp/5001
 ipfs config Addresses.Gateway /ip4/0.0.0.0/tcp/8080
 ipfs config --json Swarm.ConnMgr.HighWater 2000
 ipfs config --json Datastore.BloomFilterSize 1048576
-ipfs config Datastore.StorageMax 100GB
+ipfs config Datastore.StorageMax {{ .StorageMax }}GB
 
 chown -R ipfs: /data/ipfs
 `
 )
 
 func (r *IpfsReconciler) configMapScripts(m *clusterv1alpha1.Ipfs, cm *corev1.ConfigMap) (controllerutil.MutateFn, string) {
 	cmName := "ipfs-cluster-scripts-" + m.Name
+	configureTmpl, _ := tmpl.New("configureIpfs").Parse(configureIpfs)
+	var storageMaxGB string
+	parsed, err := resource.ParseQuantity(m.Spec.IpfsStorage)
+	if err != nil {
+		storageMaxGB = "100"
+	} else {
+		sizei64, _ := parsed.AsInt64()
```

Comment:
Be careful when using this, it seems like it may return `0, false` when it cannot easily convert to an integer.